Composer/packages/extensions/adaptive-form/src/components/SchemaField.tsx

```diff
@@ -34,9 +34,10 @@ const SchemaField: React.FC<FieldProps> = props => {
     ...rest
   } = props;
   const pluginConfig = usePluginConfig();
+
   const schema = resolveRef(baseSchema, definitions);
   const uiOptions = {
-    ...getUISchema(schema, pluginConfig.formSchema),
+    ...getUIOptions(schema, pluginConfig.formSchema, pluginConfig.roleSchema[schema.$role || '']),
```

Comment:
`pluginConfig.roleSchema[schema.$role || '']` should happen inside of `getUIOptions` just like formSchema